README.md

```diff
@@ -101,11 +111,11 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
+[x] `$(".post").css("background", "peachpuff")`
 [] `$(".post").innerHTML`
-[] `$(".post").html()`
-[] `document.getElementsByClassName("post")[0].innerHTML`
-[] `document.getElementsByClassName("post").innerHTML`
+[x] `$(".post").html()`
+[x] `document.getElementsByClassName("post")[0].innerHTML`
+[x] `document.getElementsByClassName("post").innerHTML`
 ```
```

Comment:
`document.getElementsByClassName("post").innerHTML` will not work because `.getElementsByClassName` will always return an array of DOM objects which we have to index in order to call `.innerHTML`